test/usage.cjs

```diff
@@ -2602,9 +2602,10 @@ describe('usage tests', () => {
       r.logs[0]
         .split('\n')
         .should.deep.equal([
-          'usage',
+          'usage [command]',
```

Comment:
@OsmanAltun reading both the linked bugs, I'm still a little confused about what bug this is actually addressing.

It would help me see this work over the finish line if you could:

1. add a test that explains the bug you're attempting to address.
2. update the title of the PR, to explain the specific bug fix.

I think there's a systemic bug with how default command is handled, which might be the root cause of both the bugs you've linked, but I'm not sure that the correct solution.

Having the tests, and a clear description of the bug fix, would help me better understand.